src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,19 @@
 import React from 'react';
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+import { MovieCard } from '../MovieCard/index';
+import { Movie } from '../../types/Movie';
+
+type Props = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies }) => (
+  <>
+    <div className="movies">
+      {movies.map((movie) => (
```

Comment:
```suggestion
      {movies.map((movie) => (
```
You have forgotten about the `key` attribute